### ISSUE TITLE
1.13.0 - Not all artifacts in maven central

### ISSUE THREAD

**Issue by benmadore-glassdoor** [detekt/detekt#3062]
```
   > Could not find io.gitlab.arturbosch.detekt:detekt-tooling:1.13.0.
     Required by:
         project :web-app > io.gitlab.arturbosch.detekt:detekt-cli:1.13.0
   > Could not find io.gitlab.arturbosch.detekt:detekt-parser:1.13.0.
     Required by:
         project :web-app > io.gitlab.arturbosch.detekt:detekt-cli:1.13.0
   > Could not find io.gitlab.arturbosch.detekt:detekt-core:1.13.0.
     Required by:
         project :web-app > io.gitlab.arturbosch.detekt:detekt-cli:1.13.0
   > Could not find io.gitlab.arturbosch.detekt:detekt-rules:1.13.0.
     Required by:
         project :web-app > io.gitlab.arturbosch.detekt:detekt-cli:1.13.0
```

**Comment by arturbosch** [comment]
See https://github.com/detekt/detekt/pull/3060 hopefully jcenter will resync everything.

**Comment by arturbosch** [comment]
I will spend some time in setting up direct publishing to maven central.
Bintray is highly unreliable for us since we upload checksums and signatures.
Could take some days.

**Comment by kenyee** [comment]
also:
Could not find io.gitlab.arturbosch.detekt:detekt-rules:1.13.0

**Comment by JavierSegoviaCordoba** [comment]
Also detekt-formatting

**Comment by cortinico** [comment]
> I will spend some time in setting up direct publishing to maven central.

Have you worked on this @arturbosch? Otherwise I can pick it up

**Comment by arturbosch** [comment]
> > I will spend some time in setting up direct publishing to maven central.
> 
> Have you worked on this @arturbosch? Otherwise I can pick it up

Yes, I had already set it up. Thanks for the offer!
I just published 1.13.1 to maven central directly.
It should be up soon.
